tests/get_object_or_404/tests.py

```diff
@@ -76,6 +77,26 @@ def test_get_object_or_404(self):
             [article]
         )
 
+        # get_object_or_404 can accept args
+        self.assertEqual(
+            get_object_or_404(
+                Article,
+                Q(title__startswith="Run") | Q(title__startswith="Walk"),
+                authors__name__contains="Brave"
+            ),
+            article
```

Comment:
Use single quotes and add trailing commas,
```suggestion
            get_object_or_404(
                Article,
                Q(title__startswith='Run') | Q(title__startswith='Walk'),
                authors__name__contains='Brave',
            ),
            article,
```